Homework_QAA/home_work/page_objects/all_courses_page/all_courses_page.py

```diff
@@ -0,0 +1,76 @@
+from selenium.webdriver.common.by import By
+
+from Homework_QAA.home_work.page_objects.course_page.course_page import CoursePage
+from Homework_QAA.home_work.utilities.web_ui.base_page import BasePage
+
+
+class AllCourses(BasePage):
+    def __init__(self, driver):
+        self._driver = driver
+        super().__init__(driver)
+
+    __user_avatar = (By.XPATH, "//img[@class='header__user-avatar']")
+    __search_field = (By.XPATH, '//label//input[@name="q"]')  # only after login
+    __second_page = (By.XPATH, "//li/a[text()='2']")
+    __collections = (By.CSS_SELECTOR, 'section[class="header__logo header__logo___2909e"]')
+    __previous_page_button = (By.XPATH, '//li//i[@class="fa fa-chevron-left"]')
+    __next_page_button = (By.XPATH, '//*[@aria-label="Next page"]')
+    __first_course_item = (By.CSS_SELECTOR, 'div ul:first-child li:first-child a[tabindex="0"] header img')
+    __drop_down_menu = (By.XPATH, '//li[class="dropdown header__nav-item"]')  # appears after login
+    __logout_button = (By.CSS_SELECTOR, 'ul[class="dropdown__menu"]:last-child a[href="/users/sign_out"]')  # appears after login
+    __search_title = (By.CSS_SELECTOR, 'h2[class="products__title"]')  # only after seach
+    __more_courses = (By.XPATH, "//section//a[text()='View more courses']") #button appears only before login
+    __first_title = (By.XPATH, "//header/h2[text()='Worldclass Automation Training']")
+
+
+    def user_avatar_is_visible(self):
```

Comment:
add annotation ->bool. Please add return annotations to all functions which returns smth.
Also according to PEP8 functions names which returns bool should start with is_
Please fix it in all places